integration_tests/python_modules/dagster-k8s-test-infra/dagster_k8s_test_infra/integration_utils.py

```diff
@@ -140,22 +141,22 @@ def _execute_query_over_graphql(dagit_url, query, variables):
 
 
 def launch_run_over_graphql(
-    dagit_url,
-    run_config,
-    pipeline_name,
-    repository_name="demo_execution_repo",
-    code_location_name="user-code-deployment-1",
-    solid_selection=None,
-    tags=None,
-):
+    dagit_url: str,
+    run_config: Mapping[str, Any],
+    job_name: str,
+    repository_name: str = "demo_execution_repo",
+    code_location_name: str = "user-code-deployment-1",
+    solid_selection: Optional[Sequence[str]] = None,
+    tags: Optional[Mapping[str, str]] = None,
+) -> str:
     tags = tags or {}
     variables = json.dumps(
         {
             "executionParams": {
                 "selector": {
                     "repositoryLocationName": code_location_name,
                     "repositoryName": repository_name,
-                    "pipelineName": pipeline_name,
+                    "pipelineName": job_name,
```

Comment:
does pipelineName need to get kept around here for backcompat purposes?

Reply:
I didn't touch the GQL schema in general, since that's a whole separate project requiring updates to the queries used in dagit